.gitignore

```diff
@@ -92,19 +92,19 @@ ENV/
 .ropeproject
 
 *.DS_Store
-.d3a/
+.gsy-e/
```

Comment:
Are we using `gsy-e` or `gsy_e` for folders? For example, you're using `gsy_e` below.

Reply:
`gsy-e`is the repo name as well as cli commands

`gsy_e` is the main package

Reply:
As far as I see this can be deleted though. None of our software creates a .d3a/ folder, and we do not have a specific purpose for this. I would recommend to get rid of this line, either before or after the merge of this PR. 
FYI, gsy_e would have the effect of git to ignore all our codebase :D

Reply:
Will adapt later, in order to not delay anymore